.github/CODEOWNERS

```diff
@@ -0,0 +1,7 @@
+
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+# these people will be requested for
+# review when someone opens a pull request.
+
+*       @maggief5507 @Caden740 @morganjiang1 @char-ly1 @RobertTheBlair @isaacngkh @No767  
```

Comment:
These 7 people will be automatically added to PRs, and are required for the approval